src/main/kotlin/racingcar/RacingCarApplication.kt

```diff
@@ -0,0 +1,23 @@
+package racingcar
+
+import racingcar.controller.CarController
+import racingcar.domain.Car
+import racingcar.view.InputView
+import racingcar.view.ResultView
+
+fun main() {
+    val countOfCars = InputView.inputCountOfCars()
+    val countOfTry = InputView.inputCountOfTry()
+
+    val cars = mutableListOf<Car>()
+
+    for (i in 0 until countOfCars step 1) {
+        cars.add(Car())
+    }
```

Comment:
```suggestion
    val cars = List(countOfCars) {
        Car()
    }
```

1. 코틀린은 가변보다는 불변을 더 선호하는 언어입니다. 상태에 대한 부분은 가변으로, 그 외의 것에는 불변을 지켜보는게 코드 안전성으로 좋습니다.
2. List 함수를 사용한다면 위와 같이도 표현이 가능합니다.